tensorflow/c/c_api.cc

```diff
@@ -1344,7 +1344,8 @@ void TF_OperationGetAttrString(TF_Operation* oper, const char* attr_name,
         InvalidArgument("Attribute '", attr_name, "' is not a string");
     return;
   }
-  if (max_length <= 0) {
+  if (max_length == 0) {
+    InvalidArgument("Attribute '", max_length, "' is zero");
```

Comment:
This is not the correct way to use `InvalidArgument` -- it returns a value.  And what if `max_length` is less than 0?

Reply:
Thanks Sanjoy for the comment. max_length is size_t (unsigned int), hence it can't be less than 0. So, ine worst case, if its equal to zero, we can log an error/raise an exception.

How about this: 
  if (max_length == 0) { 
    status->status = InvalidArgument("Attribute '", max_length, "' is zero");
    return;
  }